components/SiteNavigation.vue

```diff
@@ -5,13 +5,14 @@
       :key="item.title"
     >
       <Disclosure
+        v-if="item.children"
         v-slot="{ open }"
         as="div"
-        :default-open="currentPath.startsWith(item._path)"
+        :default-open="currentPath.startsWith(item.path)"
```

Comment:
@Ray2126 @tom-blake FYI I there is a bug here.
When you go to https://docs.centrapay.com/guides/ the button for the Disclosure is facing down like it is open but it is not open.
<img width="312" alt="Screenshot 2023-01-20 at 11 23 57 AM" src="https://user-images.githubusercontent.com/64108933/213575813-511a9165-49c1-4339-ab69-e902d44c09d0.png">
farmlands-pos-integration/